libs/docs/src/docs/2.2.1-alpha/overriding-pod-and-container-attributes.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Extending kubernetes resources
+description: Extending kubernetes resources
+---
+
+This section describes how you can extend Kubernetes resources for container components through `pod-overrides` and `container-overrides` attributes.
+
+## container-overrides
+
+`container-overrides` attributes allow you to override properties of a container in a pod spec such as `securityContext` and `resources`. However, it restricts from overriding properties such as `image`, `name`, `ports`, `env`, `volumesMounts`, `command`, and `args`.
```

Comment:
Have other implementations, like in DevWorkspace Operator, the same restrictions?

If not, than it needs to be mentioned

Reply:
At the time of implementation, DWO had the same restriction. I'll check again and update the doc.